mscore/palette/paletteworkspace.cpp

```diff
@@ -589,6 +589,38 @@ PaletteWorkspace::PaletteWorkspace(PaletteTreeModel* user, PaletteTreeModel* mas
       {
       if (userPalette)
             connect(userPalette, &PaletteTreeModel::treeChanged, this, &PaletteWorkspace::userPaletteChanged);
+
+      preferencesListenerID = preferences.addOnSetListener([this](const QString& key, const QVariant& value) {
+            if (key == PREF_APP_USESINGLEPALETTE)
+                  emit singlePaletteChanged();
```

Comment:
For QML purposes, it's sometimes better to pass the changed value in the signal, e.g. `singlePaletteChanged(value.toBool())`, but existing signals in the MuseScore codebase are inconsistent in that regard.

Reply:
I read different opinions about this. Some people say that you should pass the value in the signal, others say that you definitely shouldn't. Anyway, I'm not sure how to tell Qml that it should take the value from the signal. Or does it automatically, when the signal has a parameter?